qemu/tests/cfg/netkvm_log_check.cfg

```diff
@@ -0,0 +1,12 @@
+- netkvm_log_check:
+    virt_test_type = qemu
+    type = netkvm_log_check
+    only q35
+    only Windows
+    only virtio_net
+    os_type = Windows
```

Comment:
No need to define `os_type`, we can get the value directly via `params.get("os_type")` or `params["os_type"]`